src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1983,25 +1982,30 @@ def _get_static_info(self) -> dict[str, Any]:
             )
         return self._static_info
 
-    def create_instance(self) -> "PyCommandArguments":
+    def _get_create_instance_args(self):
         """Create a command instance."""
         try:
             static_info = self._get_static_info()
             id = self._create_command_arguments()
-            return PyCommandArguments(
+            return [
                 self.service,
                 self.rules,
                 self.command,
                 self.path.copy(),
                 id,
                 static_info.get("args"),
-            )
+            ]
         # Possible error thrown from the grpc layer
         except (RuntimeError, ValueError):
             logger.warning(
                 "Create command arguments object is available from 23.1 onwards"
             )
 
+    def create_instance(self) -> "PyCommandArguments":
```

Comment:
This is overwrtten in the generated explicit classes to return the object of explicit type. Minimized the implementation to reduce codegen.